.github/workflows/python-test.yml

```diff
@@ -1,12 +1,7 @@
 name: Python test
 
 # Controls when the workflow will run
-on:
-  # Triggers the workflow on push or pull request events but only for the "master" branch
-  push:
-    branches: [ "master" ]
-  pull_request:
-    branches: [ "master" ]
+on: [push]
```

Comment:
why this change? Checks on pull requests are very nice

Reply:
I wanted to check on every push, at least for the duration that we are working on this PR. Why don't we test on every push anyways?

Reply:
Ah i misunterstood this I thought it was replaced by only push to main, but it is every push. Ok!